corehq/apps/integration/kyc/views.py

```diff
@@ -64,3 +64,20 @@ def post(self, request, *args, **kwargs):
             'show_success': show_success,
         }
         return self.render_htmx_partial_response(request, self.form_template_partial_name, context)
+
+
+@method_decorator(use_bootstrap5, name='dispatch')
+@method_decorator(toggles.KYC_VERIFICATION.required_decorator(), name='dispatch')
+class KycVerificationReportView(BaseDomainView):
+    urlname = 'kyc_verify'
+    template_name = 'kyc/kyc_verify_report.html'
+    section_name = _("Data")
```

Comment:
Nit: use single quotes
```suggestion
section_name = _('Data')
```

Reply:
(because the other strings use them and the quotes looks out of place)

Reply:
Addressed in [64ed0c4](https://github.com/dimagi/commcare-hq/pull/35765/commits/64ed0c44d6d6ede3bba9c2a37103e2a9fe0db8a3).